std_msgs/Header header
sensor_msgs/NavSatFix[] coordinates
bool repeat
uint8 repeat_count # 0 means infinite loop.